src/content/docs/cypher/query-clauses/call.md

```diff
@@ -27,20 +27,22 @@ The following tables lists the built-in schema functions you can use with the `C
 | `property id` | Internal identifier of the property within table | INT64 |
 | `name` | name of the property | STRING |
 | `type` | data type of the property | STRING |
+| `default expression` | default expression of the property | STRING |
```

Comment:
This description is a unclear - what does "default expression of the property" mean?